fireworks_poe_bot/fastapi_poe/base.py

```diff
@@ -275,6 +275,11 @@ def find_bot(account: str, model: str) -> PoeBot:
             raise HTTPException(status_code=404, detail=f"Bot {bot_fqn} not found")
         return bots[bot_fqn]
 
+    @app.get("/")
+    async def index() -> Response:
+        # Default endpoint for health checks
+        return HTMLResponse("It works!")
+
```

Comment:
I mean if it is just health check then you should just do 200 lol

Reply:
consider it an easter egg lol